libs/ledger-live-common/src/network-troubleshooting/index.ts

```diff
@@ -0,0 +1,167 @@
+import { WebsocketConnectionError } from "@ledgerhq/errors";
+import axios from "axios";
+import WS from "isomorphic-ws";
+import { Observable } from "rxjs";
+import { getEnv } from "../env";
+import announcementsApi from "../notifications/AnnouncementProvider/api/api";
+import serviceStatusApi from "../notifications/ServiceStatusProvider/api/api";
+
+type TroubleshootStatus = {
+  title: string;
+  technicalDescription: string;
+  status: "success" | "error" | "loading";
+  error?: string;
+};
+
+type Troubleshoot = {
+  title: string;
+  technicalDescription: string;
+  job: Promise<unknown>;
+};
+
+// Run all checks and return. each troubleshoot have a promise that suceed if the underlying job worked.
+export function troubleshoot(): Troubleshoot[] {
+  // TODO in future, we can delegate a "troubleshoot" per coin implementation.
+  return [
+    {
+      title: "My Ledger services (scriptrunner)",
+      ...websocketConnects(
+        `${getEnv(
+          "BASE_SOCKET_URL"
+        )}/apps/list?targetId=856686596&perso=perso_11&livecommonversion=27.7.2` // TODO have a dummy echo endpoint
```

Comment:
I see a `livecommonversion=27.7.2` in the query params, will this value need to be updated each time a new LLC version is released?

Reply:
for now it's hardcoded, i just need an endpoint that works, it's not really important what this value is, but i think this field may be required (or may not be, not important)

Reply:
(in parallel, i'm asking backend to had such "health" wss version)